java-checks/src/test/java/org/sonar/java/checks/LeftCurlyBraceEndLineCheckTest.java

```diff
@@ -41,6 +41,14 @@ void java_17() {
       .verifyIssues();
   }
 
+  @Test
+  void test_record() {
+    CheckVerifier.newVerifier()
+      .onFile(TestUtils.mainCodeSourcesPath("checks/LeftCurlyBraceEndLineCheck_record.java"))
+      .withCheck(new LeftCurlyBraceEndLineCheck())
+      .verifyNoIssues();
```

Comment:
It would be nice to also check noncompliant code with records in the same file.

```suggestion
     .verifyIssues();
```
And add this code at the end of `java-checks-test-sources/default/src/main/java/checks/LeftCurlyBraceEndLineCheck_record.java`:
```java
record TestRecordSyntaxV3(Integer value)
{ // Noncompliant
  public TestRecordSyntaxV3(String v)
  { // Noncompliant
    this(Integer.parseInt(v));
  }
  public static final String CONST_1 = "IDK";
  public static final String CONST_2 = "IDK";
}
```